### DISCUSSION TITLE
How do i contribute my C# Repo to this?

### DISCUSSION THREAD

**Discussion by Marre246810** [open-spaced-repetition/free-spaced-repetition-scheduler#32] [Q&A]
Super new to contributing to an open soruce and just github in general. I've made a C# / Unity version of FSRS. How can i contribute?

**Comment by L-M-Sherlock** [comment]
If your C# Repo includes the unit tests and pass them, I will invite you to join our organization and you can move the repo into our org.

**Comment by Marre246810** [comment]
Alright! I have no tests, but will make one right now and get back in a day or two. 
Is it prefeered that it is in C#, or one with that uses the Unity libaries? Because i've made both.

**Reply by L-M-Sherlock** [reply]
I know nothing about C#. I guess it depends on the potential users of that repo.

**Reply by Marre246810** [reply]
I could make two different repos if that's better? 

They are pretty close, the libaries that they use are just different. Both use C#.
If you have some intermediate understanding you should be able to translate one to the other.

The only real difference is how the projects are setup.
